js-miniapp-bridge/bridge.js

```diff
@@ -0,0 +1,42 @@
+var messageQueue = []
+window.MiniAppBridge = {};
+var isPlatform = {
+    Android: function() {
+        return navigator.userAgent.match(/Android/i);
+    },
+    iOS: function() {
+        return navigator.userAgent.match(/iPhone|iPad|iPod/i);
+    }
+};
+
+MiniAppBridge.exec = function(action, onSuccess, onError) {
```

Comment:
As we were talking, let's put some docs as well, please.

Reply:
Done